### ISSUE TITLE
Reformatting of XML comment lists

### ISSUE THREAD

**Issue by flamewave** [codecadwallader/codemaid#221]
**Issue:**
If you have XML comment lists (for example, in the `<remarks>` section), when cleaning up the code, the XML comment list formatting gets destroyed.

For example, if you had the following XML comment list in your remarks section:
```
/// <remarks>
///   <list type="table">
///     <listheader>
///       <term>Term</term>
///       <description>Description</description>
///     </listheader>
///     <item>
///       <term>Test item 1</term>
///       <description>Test description 1</description>
///     </item>
///   </list>
/// </remarks>
```

it gets reformatted upon code cleanup to be all on one line (or multiple if long enough to wrap as per max line length settings):

```
/// <remarks>
/// <list type="table"><listheader><term>Term</term>description>Description</description>
/// </listheader><item><term>Test item 1</term><description>Test description 1</description>
/// </item></list>
/// </remarks>
```

This reduces the readability of these lists (when looking at them in code) greatly.

**Expected Behavior:**
When cleaning up code with XML list comments, they should be formatted appropriately (similar to my first example above). If the text within a `<term>` or `<description>` node is longer than the configured line length, it should wrap just that node as it does with other XML comment nodes.

**Comment by flamewave** [comment]
I should add, this is happening in CodeMaid version 0.9.0

**Comment by codecadwallader** [comment]
The latest beta has some fixes specifically for XML list comments.  Can you give that a shot?  Note: it has some other outstanding issues that are being investigated so you may want to uninstall the beta and reinstall v0.9.0 after testing.

https://github.com/codecadwallader/codemaid/releases/tag/v0.9.0.1-beta

**Comment by flamewave** [comment]
Looks like it works in the beta. Only minor complaint is that it doesn't indent the elements, but that's more of a personal preference thing that perhaps you can expand on later.

I did notice the issues you were talking about and will be reverting back to v0.9.0. There was one that I didn't see a report for though, where it is inserting an extra `///` with tabs/spaces on the line after closing tags that are on their own line. Let me know if you want me to create a new issue for this if you don't already have one somewhere that I didn't find.

**Comment by codecadwallader** [comment]
Thanks for testing it out, glad to hear it covers your scenario.

I did see the trailing line issue as well and expect that will be resolved as part of #215
